Bottom: 8ba682a0c3c33065ac98b25eb7d89e6bd3692e6a
Top:    4295d28a925ab575732e7acc2019c109ebf2fba1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:57:31 +0000

hid/batch: Add some missing header files for prototypes


---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index 8d53d8c..e4f3311 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -12,6 +12,8 @@
 #include "global.h"
 #include "hid.h"
 #include "data.h"
+#include "misc.h"
+#include "hid.h"
 
 #include "hid/common/actions.h"
 #include "hid/common/hidinit.h"
